keel-core/src/main/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunner.kt

```diff
@@ -91,8 +91,11 @@ class VerificationRunner(
     }
   }
 
+  /**
+   * `true` if any of the statuses is [PENDING], `false` if none are or the collection is empty.
+   */
   private val Collection<Pair<*, ConstraintStatus?>>.anyStillRunning: Boolean
-    get() = !none { (_, status) -> status == PENDING }
+    get() = isNotEmpty() && any { (_, status) -> status == PENDING }
```

Comment:
A wise man once noted that `any` will return false if the collection is empty, so the `isNotEmpty` isn't necessary.

https://github.com/spinnaker/keel/pull/1834#discussion_r585986054

Reply:
Dammit. Why can't I get a simple bit of logic right. At least that explains why it didn't break a test.